### PR TITLE
Gidk 1dcrrf datepicker test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    border: 1px solid #ccc;
+    z-index: 999!important;
+}
+.drp-selected{
+    display: none;
+}
+.daterangepicker:after {
+    top: -6px;
+    border-right: 6px solid transparent;
+    border-bottom: 6px solid #fff;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss:27]: use color variable

File: apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+.daterangepicker th.month {
+    color: $nobel;
+}
+.daterangepicker th{
+    color: $dark-gray;
+}
+th.prev.available span {
+    width: 10px;
+    height: 10px;
+    border-left: 2px solid #666;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss:67]: use color variable

File: apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    display: flex;
+    border-bottom: 2px solid #666;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss:69]: use color variable

File: apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    transform: rotate(45deg);
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+    margin-left: 8px;
+}
+th.next.available span {
+    width: 10px;
+    height: 10px;
+    border-left: 2px solid #666;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss:79]: use color variable

File: apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    display: flex;
+    border-bottom: 2px solid #666;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss:81]: use color variable

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 6]
```diff
@@ -1968,6 +1969,19 @@ header .navbar {
 
 /*btn*/
 
+.btn-pink {
+    background: #01a9f4;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:1973]: use color variable everywhere

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 7]
```diff
@@ -1968,6 +1969,19 @@ header .navbar {
...
+    border: 1px solid #01a9f4;
+    color: #fff;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:1975]: use color variable

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 8]
```diff
@@ -1968,6 +1969,19 @@ header .navbar {
 
 /*btn*/
 
+.btn-pink {
+    background: #01a9f4;
+    border: 1px solid #01a9f4;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:1974]: use color variable

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 9]
```diff
@@ -1968,6 +1969,19 @@ header .navbar {
...
+    color: #fff;
+    &:hover{
+        color:#fff;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:1977]: use color variable

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 9 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: All color name updated with color var

File: apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+        padding: 6px !important;
+    }
+}
+.daterangepicker .calendar-table{
+    padding-right: 12px;
+}
+.daterangepicker{
+    box-shadow: 0 3px 28px 0 rgba(0,0,0,.14);
+    color: $dark-gray;
+    border: 1px solid #ccc;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss:18]: use color var

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss (hunk 10 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/ngx-bs-datepicker.scss]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**